components/theme.ts

```diff
@@ -49,5 +49,17 @@ export const ThemeProvider = EmotionThemeProvider as React.ProviderExoticCompone
 >;
 export const useTheme = emotionUseTheme as () => Theme;
 
+export function themeGet(type: keyof Theme, name: keyof Theme['sizes']) {
```

Comment:
NP: technically, this `name` param should probably have been typed as `string` rather than `keyof Theme['sizes']` 🤷‍♂️ .

Reply:
And perhaps it doesn't need exporting?

Reply:
It's good to have it exported, I use it already in a few places to get zIndexes. It's kind of the generic helper do get theme data we don't use often...